src/zabapgit_services.prog.abap

```diff
@@ -2,8 +2,9 @@
 *&  Include           ZABAPGIT_SERVICES
 *&---------------------------------------------------------------------*
 
+INCLUDE zabapgit_services_git.
+INCLUDE zabapgit_services_trans_branch.
```

Comment:
I would like to keep it out of the git include, as this is not a raw git command

I think it is okay to have it in its own include, perhaps rename it to something else?